src/components/Permissions/DataList.jsx

```diff
@@ -1,15 +1,116 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
 import Page from 'components/Page'
 import PageTitle from 'components/PageTitle'
-import { useI18n } from 'cozy-ui/transpiled/react/I18n'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
+import ListItem from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItem'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import ListItemIcon, {
+  mediumSize
+} from 'cozy-ui/transpiled/react/MuiCozyTheme/ListItemIcon'
+import Divider from 'cozy-ui/transpiled/react/MuiCozyTheme/Divider'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import Spinner from 'cozy-ui/transpiled/react/Spinner'
+import { APPS_DOCTYPE, KONNECTORS_DOCTYPE } from 'doctypes'
+import withAllLocales from 'lib/withAllLocales'
+import { getPermissionIconName } from 'containers/helpers/permissionsHelper'
+import CozyClient, {
+  Q,
+  useQuery,
+  isQueryLoading,
+  hasQueryBeenLoaded
+} from 'cozy-client'
 
-const DataList = () => {
-  const { t } = useI18n()
+const THIRTY_SECONDS = 30 * 1000
+const DataList = ({ t }) => {
+  const queryResultApps = useQuery(Q(APPS_DOCTYPE), {
+    as: APPS_DOCTYPE,
+    fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
+  })
+  const queryResultKonnectors = useQuery(Q(KONNECTORS_DOCTYPE), {
+    as: KONNECTORS_DOCTYPE,
+    fetchPolicy: CozyClient.fetchPolicies.olderThan(THIRTY_SECONDS)
+  })
+
+  const completePermission = () => {
+    const hiddenSlugs = ['home', 'store', 'settings']
+    let queryResultSlugs = []
+    let permissionsToDisplay = {}
+    if (
+      queryResultApps?.data?.length > 0 &&
+      queryResultKonnectors?.data?.length > 0
+    ) {
+      queryResultSlugs = [
+        ...queryResultApps.data,
+        ...queryResultKonnectors.data
+      ].filter(({ slug }) => !hiddenSlugs.includes(slug))
+
+      for (let { slug, permissions } of queryResultSlugs) {
+        Object.values(permissions).map(({ type }) => {
+          if (!permissionsToDisplay[type]) {
+            permissionsToDisplay[type] = []
+          }
+          permissionsToDisplay[type].push(slug)
+        })
+      }
+    }
+    return permissionsToDisplay
+  }
+  const permissionsToDisplay = completePermission()
+  const sortPermissionsByName = permissionsToDisplay => {
+    return Object.entries(permissionsToDisplay).sort((a, b) => {
+      return t('CozyClient.Permissions.' + a[0])
```

Comment:
This is not `CozyClient` permission. It can be anything. CozyClient is just one of many clients. 

We're talking about the access to your cozy. `CozyPermissions` maybe?